<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoundSettingsPage</class>
 <widget class="QWidget" name="SoundSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="driverSndLabel">
       <property name="text">
        <string>Driver</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="driverSndComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Auto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>OpenAL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>XAudio2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FMod</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DSound</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="autoLoadSampleCheckBox">
     <property name="text">
      <string>SoundAutoLoadSample</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QCheckBox" name="enableOccludeObstructCheckBox">
     <property name="text">
      <string>SoundEnableOccludeObstruct</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="enableReverbCheckBox">
     <property name="text">
      <string>SoundEnableReverb</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="manualRolloffCheckBox">
     <property name="text">
      <string>SoundManualRolloff</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="forceSoftwareCheckBox">
     <property name="text">
      <string>SoundForceSoftware</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QCheckBox" name="useADPCMCheckBox">
     <property name="text">
      <string>SoundUseADPCM</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="maxTrackLabel">
       <property name="text">
        <string>SoundMaxTrack</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="maxTrackSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="specialValueText">
        <string notr="true"/>
       </property>
       <property name="suffix">
        <string notr="true"/>
       </property>
       <property name="prefix">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>SoundPackedSheetPath</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="soundSheetPathLineEdit"/>
   </item>
   <item row="8" column="2">
    <widget class="QToolButton" name="soundSheetPathButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>SoundSamplePath</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="soundSamplePathLineEdit"/>
   </item>
   <item row="9" column="2">
    <widget class="QToolButton" name="soundSamplePathButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>83</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../object_viewer_qt.qrc"/>
 </resources>
 <connections/>
</ui>
